xml/Microsoft.Extensions.Primitives/StringValues.xml

```diff
@@ -1511,9 +1549,10 @@
         <Parameter Name="item" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="item">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="item">The object to remove from the <see cref="T:System.Collections.Generic.ICollection`1" />.</param>
```

Comment:
We already know the type, so natural language might be better here:
```suggestion
        <param name="item">The object to remove from the collection.</param>
```

Reply:
There are instances in System.Collections.Generic that already use the cref.
How about I keep as is for consistency?